main.go

```diff
@@ -391,7 +394,19 @@ func main() {
 
 		}
 
-		if err = otelv1beta1.SetupCollectorWebhook(mgr, cfg, reviewer, crdMetrics); err != nil {
+		bv := func(collector otelv1beta1.OpenTelemetryCollector) (admission.Warnings, error) {
```

Comment:
It looks like we're not using the admission.Warnings at all here, should we move the errors to the warnings, or just get rid of the warnings and only do errors?